code/src/App.js

```diff
@@ -1,9 +1,109 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { Start } from 'components/Start';
+import { Sleep } from 'components/Sleep';
+import { Food } from 'components/Food';
+import { Activity } from 'components/Activity';
+import { Summary } from 'components/Summary';
+import { Button } from 'components/Button';
 
 export const App = () => {
+  const [counter, setCounter] = useState(0)
+  const [userSleepHabit, setUserSleepHabit] = useState('')
+  const [userFoodHabit, setUserFoodHabit] = useState('')
+  const [userActivity, setUserActivity] = useState('')
+
+  const handleCounterButtonClick = (shouldIncrease) => {
+    if (shouldIncrease && counter < 4) {
+      setCounter(counter + 1)
+    } else if (!shouldIncrease && counter > 0) {
+      setCounter(counter - 1)
+    }
+  }
+  const refreshPage = () => {
+    window.location.reload(false);
+  }
+  const handleSubmit = () => {
+    window.alert('Thanks for doing our survey!')
+  }
+
   return (
-    <div>
-      Find me in src/app.js!
+    <div className="container">
+      <div className="survey-box">
+        {counter === 0 && (
+          <>
+            <Start />
+            <Button
+              btnText="Begin!"
+              btnClickFunction={() => handleCounterButtonClick(true)} />
+          </>
+        )}
+        {counter === 1 && (
+          <>
+            <Sleep
+              sleepHabit={userSleepHabit}
+              setSleepHabit={setUserSleepHabit} />
+
+            <div className="button-group">
+              <Button
+                btnText="Back"
+                btnClickFunction={() => handleCounterButtonClick(false)} />
```

Comment:
Nice idea to have a button that takes you back to the previous question. Easy solution that does a lot for accessibility!